docs/src/Theory/Atmos/Microphysics.md

```diff
@@ -678,32 +681,35 @@ If ``T > T_{freeze}``:
 
 ## Coupling to the state variables
 
-### Rain
+### Warm Rain
 
-The source of rain water ``\mathcal{S}_{q_{rai}}`` is a sum of the
+The source of rain ``\mathcal{S}_{q_{rai}}`` is a sum of the
   [autoconversion](https://clima.github.io/ClimateMachine.jl/latest/Theory/Atmos/Microphysics/#Rain-autoconversion),
   [accretion](https://clima.github.io/ClimateMachine.jl/latest/Theory/Atmos/Microphysics/#Accretion), and
   [rain evaporation](https://clima.github.io/ClimateMachine.jl/latest/Theory/Atmos/Microphysics/#Rain-evaporation-and-snow-sublimation)
-  sources.
-The sink of total water is equal to the source of rain water:
+  processes.
+The sink of total water is equal to the source of rain:
   ``\mathcal{S}_{q_{tot}}`` = -``\mathcal{S}_{q_{rai}}``.
-The sink of liquid water is equal to the source of rain water
-  from autoconversion and accretion:
-  ``\mathcal{S}_{q_{liq}}`` = - ``\left.\mathcal{S}_{q_{rai}} \right|_{acnv}`` - ``\left. \mathcal{S}_{q_{rai}} \right|_{accr}``.
+The sink of cloud liquid water is the sum of rain autoconversion
+  and rain accretion processes.
 Following the conservation equations for
-[moisture](https://clima.github.io/ClimateMachine.jl/latest/Theory/Atmos/AtmosModel/#Moisture)
-and [mass](https://clima.github.io/ClimateMachine.jl/latest/Theory/Atmos/AtmosModel/#Mass),
+[mass](https://clima.github.io/ClimateMachine.jl/latest/Theory/Atmos/AtmosModel/#Mass),
+[moisture](https://clima.github.io/ClimateMachine.jl/latest/Theory/Atmos/AtmosModel/#Moisture), and
+[precipitation](https://clima.github.io/ClimateMachine.jl/latest/Theory/Atmos/AtmosModel/#Precipitating-Species)
 the ``\mathcal{S}_{q_{tot}}`` sink has to be multiplied by ``\rho`` before
   adding it as one of the sink terms to both ``\rho`` and ``\rho q_{tot}``
   state variables.
-Similarly, the ``\mathcal{S}_{q_{liq}}`` sink has to be multiplied by ``\rho``
-  before adding it as one of the sink terms to ``\rho q_{liq}`` state variable.
+The ``\mathcal{S}_{q_{liq}}`` sink has to be multiplied by ``\rho``
+  before adding it as one of the sink terms to ``\rho q_{liq}`` state variable and
+  the ``\mathcal{S}_{q_{rai}}`` source has to be multiplied by ``\rho``
+  before adding it as one of the source terms to ``\rho q_{rai}`` state variable.
 For the conservation equation for
 [total energy](https://clima.github.io/ClimateMachine.jl/latest/Theory/Atmos/AtmosModel/#Energy),
   the sink due to removing ``q_{tot}`` is computed as:
 ```math
 \begin{equation}
-\left. \sum_{j\in\{v,l,i\}}(I_j + \Phi)  \rho C(q_j \rightarrow q_p) \right|_{precip} =
+\left. \mathcal{S}_{\rho e}} \right|_{precip} =
+  \left. \sum_{j\in\{v,l,i\}}(I_j + \Phi)  \rho C(q_j \rightarrow q_p) \right|_{precip} =
```

Comment:
Looks like this broke the latex equation

Reply:
damn. I remember adding this line thinking "not gonna compile it, its only one line, what could possibly go wrong". Well apparently everything can. I hope removing one random "}" fixes it.